consensus/ibft/src/main/java/tech/pegasys/pantheon/consensus/ibft/validation/RoundChangeCertificateValidator.java

```diff
@@ -58,7 +59,8 @@ public boolean validateRoundChangeMessagesAndEnsureTargetRoundMatchesRoot(
     final Collection<SignedData<RoundChangePayload>> roundChangeMsgs =
         roundChangeCert.getRoundChangePayloads();
 
-    if (roundChangeMsgs.size() < quorum) {
+    if (roundChangeMsgs.stream().map(SignedData::getAuthor).collect(Collectors.toSet()).size()
```

Comment:
This can be written a bit more simply: roundChangeMsgs.stream().map(SignedData::getAuthor).distinct().count()

Reply:
Nice...I didn't know about `distinct()`